crates/rpc/src/method/get_storage_proof.rs

```diff
@@ -988,6 +988,7 @@ mod tests {
         }
     }
 
+    /*
```

Comment:
The test could be fixed, right?

Reply:
Yes, it's a side effect of rework in progress, the PR was put back to draft at that point.